templates/types/streaming/fastapi/app/config.py

```diff
@@ -0,0 +1 @@
+DATA_DIR = "data"
```

Comment:
@leehuwuj where could be put this otherwise?

Reply:
It's fine to me